contracts/utils/AdminRole.sol

```diff
@@ -1,8 +1,9 @@
 pragma solidity 0.5.17;
 
 import "../openzeppelin/Ownable.sol";
+import "./AdminManagerRole.sol";
 
-contract AdminRole is Ownable {
+contract AdminRole is Ownable, AdminManagerRole {
```

Comment:
we need AdminManager only for VestingRegistry which is an exception (because admins are EOAs creating vesting contracts to avoid huge amount of multisig txs to sign on each vesting creation) and can be set and removed following current needs, but in all other cases the admin role is assigned to a multisig and should be managed by Bitocracy only. therefore there is no justification on why LiquidityMining should have this admin manager role too.  
to keep it consistent with the ticket requirements, i suggest to introduce a new role AdminRoleManaged rather than extending the existing AdminRole.  
this way we have separation of concerns and clear alternative should we ever need to use it again.